CHANGELOG.md

```diff
@@ -56,10 +56,12 @@ All notable changes to this project will be documented in this file.
     * #### Added
         * `fetch_capture_waveform()` - returns dictionary { site: data, site: data, ... }
         * `pins` is now a valid repeated capability
+        * `fetch_capture_waveform()` - returns namedtuple `Waveform(site, data)`
```

Comment:
this appears to be out of date.

Reply:
It is actually an old version of the line from two lines above.

Removed